intelmq/lib/bot.py

```diff
@@ -286,8 +289,10 @@ def acknowledge_message(self):
         self.__last_message = self.__current_message
         self.__source_pipeline.acknowledge()
 
-    def __dump_message(self, error_traceback):
-        if self.__current_message is None:
+    def _dump_message(self, error_traceback, message=None):
+        if message is None:
```

Comment:
For the ParserBot, `self.__current_message` lost it's meaning. Thus I needed to give it directly. But it would be cleaner anyway to remove the fallback to current message at all and use the parameter only. And raise a warning, if None is encountered.